Packs/Core/ReleaseNotes/3_0_6.md

```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Local Analysis alert Investigation
+
+- Updated the input `Query` to search for alerts on the endpoint by agent ID instead of host IP using the sub-playbook "Enrichment for verdict".
```

Comment:
Please change to - ....used by the sub-playbook....

Reply:
done